### DISCUSSION TITLE
Hello Elementor Meetings Mobile View - 'Any Type' menu drop-down off page

### DISCUSSION THREAD

**Discussion by RockfaceIT** [code4recovery/12-step-meeting-list#1365] [Q&A] [answered]
We are using Elementor Pro with a Child Hello Elementor Theme.  It's a fairly plain installation.

If I change the theme to twenty-twenty-four then the issue goes away.

When on a mobile device and looking at the meetings page the 'any type' drop down menu goes off the page.

URL:
https://alanonventura.rockfaceit.com/meetings/

Screenshot with hello elementor theme:
<img width="476" alt="Screen Shot 2024-02-19 at 10 41 41 PM" src="https://github.com/code4recovery/12-step-meeting-list/assets/37557944/424f8b48-1822-4d5a-8476-4805788bb7c4">

Screenshot with twenty twenty four theme:  Looks fine
<img width="485" alt="Screen Shot 2024-02-19 at 10 41 14 PM" src="https://github.com/code4recovery/12-step-meeting-list/assets/37557944/ed81b2b2-f780-40cc-b709-92ba3ef11247">

Is there some CSS I can install on Hello?

Thanks everyone!

**Comment by joshreisner** [comment]
yeah, looks like your theme is applying the CSS style `white-space: nowrap` to your buttons, which is what is causing this. if you could override that with some custom CSS that could fix the wrapping issue. 

<img width="1840" alt="Screenshot 2024-02-20 at 8 05 13 AM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/547f08e6-1c65-4b34-8e53-4a900ee7daaf">

we'll also fix it on our site - we include a lot of CSS in TSML UI to defend against interference from themes, i will add this scenario to the list https://github.com/code4recovery/tsml-ui/issues/362

**Reply by joshreisner** [reply, marked as answer]
ok! with the latest [TSML UI update](https://github.com/code4recovery/tsml-ui/pull/363) hopefully you won't need the extra CSS anymore. but it won't hurt to leave it in either

**Comment by RockfaceIT** [comment]
Perfect.  Thanks!  I added this to my customizer:

#tsml-ui button { white-space: normal; }
